demo/basic/popup/index.htm

```diff
@@ -4,7 +4,7 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-material-design/4.0.2/bootstrap-material-design.css" />
     <link rel="stylesheet" href="../../common/index.css" />
 
-    <script src="../../../node_modules/jsx-pragmatic/dist/jsx-pragmatic.js"></script>
+    <script src="https://npmcdn.com/jsx-pragmatic@2.0.20/dist/jsx-pragmatic.js"></script>
```

Comment:
At first I was going to recommend using `unpkg.com` but then I clicked on this link and realized it redirects to unpkg. TIL 😄

Reply:
Although they're the same, npmcdn was the older url for unpkg and it looks like they prefer folks to use the new url. I'm pushing up a small update to use that instead.